client/wfnews-war/src/main/angular/src/app/components/public-incident-page/incident-info-header/incident-header-panel.component.html

```diff
@@ -13,97 +13,142 @@
         <img alt="icon" (click)="back()" height="24" width="24" src="/assets/images/svg-icons/arrow-left-grey.svg">
         <span>Back</span>
       </div>
-      <div class="title-bar">
-        <div class="title">{{incident.incidentName}}
-            <span #el *ngIf="!isMobileView()" class="no-print" style="cursor: pointer;"
-              (click)="onWatchlist() ? removeFromWatchlist() : addToWatchlist()">
-              <img *ngIf="!onWatchlist()" height="20" width="20" class="bookmark-icon" src="/assets/images/svg-icons/bookmark.svg"
-              alt="bookmark" />
-              <img *ngIf="onWatchlist()" height="20" width="20" class="bookmark-icon" src="/assets/images/svg-icons/bookmark-blue.svg"
-              alt="bookmark" />
-            </span>
-        </div>
-        <div *ngIf="!isMobileView()" class="evac-cards">
-          <div class="alert-count" *ngIf="incidentEvacAlerts.length > 0">
-              <img class="icon" src="/assets/images/svg-icons/evacuation-alert.svg" alt="evacuation-alert" />
-              <span>{{ incidentEvacAlerts.length > 1 ? incidentEvacAlerts.length + ' Evacuation Alerts' : 'Evacuation Alert' }}</span>
+      <div *ngIf="incident">
+        <div class="title-bar">
+          <div class="title">{{incident.incidentName}}
+              <span #el *ngIf="!isMobileView()" class="no-print" style="cursor: pointer;"
```

Comment:
Yes, the header component can be used by both of desktop or mobile view. I think it make sense leave the isMobileView check in header component to avoid duplication, otherwise we will have to pass the value from every individual detail page components